src/components/Shell/Shell.tsx

```diff
@@ -142,8 +143,28 @@ export const Shell = ({ appNeedsUpdate, children, userPeerId }: ShellProps) => {
   }, [title])
 
   useEffect(() => {
-    setShowRoomControls(showAppBar)
-  }, [showAppBar, setShowRoomControls])
+    if (isFullscreen) {
+      document.body.requestFullscreen()
+      setShowRoomControls(false)
+      setShowAppBar(false)
+    } else {
+      if (document.exitFullscreen) {
+        document.exitFullscreen()
+      } else if (document.mozCancelFullScreen) {
+        document.mozCancelFullScreen()
+      } else if (document.webkitExitFullscreen) {
+        document.webkitExitFullscreen()
+      } else if (document.msExitFullScreen) {
+        document.msExitFullScreen()
+      }
```

Comment:
It looks like this is causing the TypeScript build to break:

```
src/components/Shell/Shell.tsx(153,27): error TS2339: Property 'mozCancelFullScreen' does not exist on type 'Document'.
src/components/Shell/Shell.tsx(154,18): error TS2339: Property 'mozCancelFullScreen' does not exist on type 'Document'.
src/components/Shell/Shell.tsx(155,27): error TS2339: Property 'webkitExitFullscreen' does not exist on type 'Document'.
src/components/Shell/Shell.tsx(156,18): error TS2339: Property 'webkitExitFullscreen' does not exist on type 'Document'.
src/components/Shell/Shell.tsx(157,27): error TS2551: Property 'msExitFullScreen' does not exist on type 'Document'. Did you mean 'exitFullscreen'?
src/components/Shell/Shell.tsx(158,18): error TS2551: Property 'msExitFullScreen' does not exist on type 'Document'. Did you mean 'exitFullscreen'?
```

The logic here seems sound, we just need keep the TypeScript compiler satisfied.